pycircleci/api.py

```diff
@@ -867,6 +902,30 @@ def project_slug(self, username, reponame, vcs_type=GITHUB):
         slug = "{0}/{1}/{2}".format(vcs_type, username, reponame)
         return slug
 
+    def owner_slug(self, username, vcs_type=GITHUB):
+        """Return owner slug as string
+
+        :param username: Org or user name.
+        :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
+
+        Returns: string ``:vcs-type/:username"``
+        """
+
+        slug = "{0}/{1}".format(vcs_type, username)
```

Comment:
I followed the existing code here, though open to using f-strings.

Reply:
I'd go with f-strings anytime. However, this is needed for backward compatibility.